src/components/StablecoinsTable.tsx

```diff
@@ -50,20 +50,18 @@ const StablecoinsTable = ({
     <Table
       variant="unstyled"
       my={8}
-      borderRadius="sm"
-      border={`1px solid ${textColor}`}
       bg="background.base"
       mb={8}
       minW="720px"
     >
-      <Thead bg="ednBackground" color="text200">
-        <Tr borderBottom={`1px solid ${textColor}`} mb="1px">
+      <Thead bg="background.highlight" color="text200">
+        <Tr mb="1px">
```

Comment:
Could someone drop the link to the Figma for this? Feel like we shouldn't have 1px adjustments

Reply:
> Could someone drop the link to the Figma for this? Feel like we shouldn't have 1px adjustments

https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System?type=design&node-id=1961%3A33521&mode=design&t=cm7YrGNCoOVawgRV-1